crates/ruff_linter/src/rules/pylint/rules/if_stmt_min_max.rs

```diff
@@ -154,21 +154,31 @@ pub(crate) fn if_stmt_min_max(checker: &mut Checker, stmt_if: &ast::StmtIf) {
         (right, left.as_ref())
     };
 
-    let replacement = format!(
-        "{} = {min_max}({}, {})",
-        checker.locator().slice(
-            parenthesized_range(
-                body_target.into(),
-                body.into(),
-                checker.comment_ranges(),
-                checker.locator().contents()
-            )
-            .unwrap_or(body_target.range())
-        ),
-        checker.locator().slice(arg1),
-        checker.locator().slice(arg2),
+    // Extract the target of the comparison
+    let cmp_target = checker.locator().slice(
+        parenthesized_range(
+            body_target.into(),
+            body.into(),
+            checker.comment_ranges(),
+            checker.locator().contents(),
+        )
+        .unwrap_or(body_target.range()),
     );
 
+    // Make sure that the first arg of the min/max method is equal
+    // to the target of the comparison
+    let first_cmp_arg: &str;
+    let second_cmp_arg: &str;
+    if checker.locator().slice(arg1) == cmp_target {
```

Comment:
Can we reuse `flip_args` from above or the `left_is_target`/`right_is_target` variables?